### PR TITLE
Update tr

### PR DESCRIPTION
**Pull request by mikropsoft**
I had to correct the translation from the beginning, so I didn't use weblate. the weblate interface is a bit complicated for me, if there is an option to upload files, I couldn't see it. please accept this.

### CODE REVIEW

**Comment by PoneyClairDeLune** [comment]
@mikrosoft Reviewing without Weblate would take quite some time. Lemme sift through it first.

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -1,981 +1,1337 @@
...
-    <string name="settings_go_log_heading" translatable="true">Günlük seviyesi</string>
-    <string name="firewall_rule_domain_blocked_univ_desc" translatable="true">Bu bağlantı, evrensel alan kurallarında <i>alan adı</i> olarak engellendiği için engellendi.<br/><br/> Bu davranışı değiştirmek için <i>DNS Yapılandırma</i> sekmesine gidin.</string>
-    <string name="lbl_auto" translatable="true">(otomatik)</string>
-    <string name="bad_config_explanation_udp_port" translatable="true">: Geçerli bir UDP bağlantı noktası numarası olmalıdır</string>
-    <string name="lbl_slow" translatable="true">yavaş</string>
-    <string name="google_dns_desc" translatable="true">Herhangi bir DNS isteğini engellemez. Google\'ın DNS uç noktasını kullanır.</string>
-    <string name="settings_allow_lan_desc" translatable="true">LAN, geri döngü, çok noktaya yayın, bağlantı yerel yollarını Rethink\'in VPN tünelinden hariç tutun.</string>
-    <string name="bad_config_reason_invalid_value" translatable="true">Geçersiz değer</string>
+    <string name="fapps_exclude_block_dialog_message" translatable="true">Hariç tut
```
Comment (PoneyClairDeLune) [inline app/src/main/res/values-tr/strings.xml:799]: MFP

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -1,981 +1,1337 @@
...
+    <string name="coming_soon_toast" translatable="true">çok yakında</string>
+    <!-- Pause Activity -->
+    <string name="pause_text" translatable="true">duraklatıldı</string>
+    <string name="pause_desc" translatable="true">Not: %1$s engellenen uygulama güvenlik duvarı ile korunmaya devam edecektir.</string>
+    <!-- Other DNS list -->
+    <string name="other_dns_list_tab1" translatable="true">DoH</string>
+    <string name="other_dns_list_tab2" translatable="true">Dnscrypt</string>
+    <string name="other_dns_list_tab3" translatable="true">Dns Proxy</string>
+    <!-- Home screen bottom sheet -->
+    <string name="hs_btm_sheet_lock_down" translatable="true">VPN açıkken modlar değiştirilemez
```
Comment (PoneyClairDeLune) [inline app/src/main/res/values-tr/strings.xml:849]: MFP

**Review by PoneyClairDeLune — COMMENTED** [review]
Just some formatting issues left. I'll try to fix everything.

**Comment by mikropsoft** [comment]
oh ok, i don't understand how i made these mistakes. i will update soon.

**Comment by mikropsoft** [comment]
really the formatter has completely corrupted the file 🤦‍♂

**Comment by PoneyClairDeLune** [comment]
> Can you check again?

@mikropsoft Just committed some minor fixes. If the CI passes this time, it should be good to go.

**Comment by mikropsoft** [comment]
> > Can you check again?
> 
> @mikropsoft Just committed some minor fixes. If the CI passes this time, it should be good to go.

If it fails, leave it to me. I will try for a few more days and if not I will close it

**Comment by PoneyClairDeLune** [comment]
Most of the errors now seems to be related to invalid Unicode/escape sequences. https://github.com/celzero/rethink-app/actions/runs/8255352645/job/22581616298?pr=1262#step:5:6700

**Comment by PoneyClairDeLune** [comment]
> I will also remove a few strings that I suspect and try again. Will this cause an error?

Maybe not. Untranslated strings should fallback to English just fine.

**Pull request closed without merging**